luminoth/train.py

```diff
@@ -16,8 +16,9 @@
 )
 
 
-def run(model_type, config_file, override_params, target='', cluster_spec=None,
-        is_chief=True, job_name=None, task_index=None, **kwargs):
+def run(model_type, dataset_type, config_file, override_params, target='',
+        cluster_spec=None, is_chief=True, job_name=None, task_index=None,
+        get_dataset=None, get_model=None, **kwargs):
```

Comment:
Use defaults for `get_dataset`, `get_model`.

Also, rename kwargs to `get_dataset_fn` and `get_model_fn` to avoid confusion.